OracleWebLogic/samples/12213-domain-home-in-image-ssl/README.md

```diff
@@ -0,0 +1,98 @@
+
+**Note:** In this sample, the WebLogic Servers are configured with a blank listen address; when running JTA transactions, you must use a DNS server to configure the listen addresses to use DNS names.
+
+## Providing the Administration Server user name and password
+
+
+**During Docker Build:** The user name and password must be supplied in the `domain_security.properties` file.  The property file is located in the directory `docker-images/OracleWebLogic/samples/12213-domain-home-in-image-ssl/properties/docker_build` in the HOST. This property file gets copied into the image directory `/u01/oracle/properties`.
+
+**During Docker Run:** The user name and password must be supplied in a `security.properties` file.  The property file is located in a `docker-images/OracleWebLogic/samples/12213-domain-home-in-image-ssl/properties/docker_run` directory in the HOST. On the Docker run command line, the `-v` option maps the property file into the image directory `/u01/oracle/properties`.
+
+
+The security property files enable the scripts to configure the correct authentication for the WebLogic Administration Server and Managed Servers. The format of the `security.properties` and `domain_security.properties` files are key=value pairs, for example:
+
+	username=myadminusername
+	password=myadminpassword
+
+**Note:** Oracle recommends that the `domain_security.properties` and `security.properties` files be deleted or secured after the container and the WebLogic Server are started so that the user name and password are not inadvertently exposed.
+
+## How to Build and Run
+At build time, the `domain.properties` file is used to pass in the Docker arguments and configuration parameters for the WebLogic domain.
+
+
+**During Docker Build:** The domain configuration parameters must be supplied in the `domain.properties` file.  This file is located in the directory `properties/docker_build` in the HOST. This property file gets copied into the image directory `/u01/oracle/properties`.
+
+
+The domain property file enables you to customize the parameters to configure the WebLogic domain. The format of the `domain.properties` are key=value pairs, for example:
+
+	ADMIN_NAME=admin-server
+	ADMIN_HOST=wlsadmin
+	MANAGED_SERVER_NAME_BASE=managed-server
+	CONFIGURED_MANAGED_SERVER_COUNT=2
+	CLUSTER_NAME=cluster-1
+	DEBUG_FLAG=true
+	PRODUCTION_MODE_ENABLED=true
+	CLUSTER_TYPE=DYNAMIC
+	CLUSTER_NAME=cluster1
+
+**NOTE:** Before invoking the build make sure you have built `oracle/weblogic:12.2.1.3-developer`.
+
+
+Under the directory `docker-images/OracleWebLogic/samples/12213-domain-home-in-image-ssl/container_scripts` find the script `setEnv.sh`. This script extracts the following Docker arguments and passes them as a `--build-arg` to the Dockerfile.
+
+
+* Domain Name:              `DOMAIN_NAME`             (default: `base_domain`)  
```

Comment:
All of your `--args` parameters are prefixed with `CUSTOM_` so either drop the `CUSTOM_` or fix the docs to match.